students/JimJenkins/session01/break_me.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+'''
+Author: Jim Jenkins (dvlupr)
+Date: 10/13/2018
+
+Each function, when called, should cause an exception to happen
+Each function should result in one of the four most common exceptions you’ll find.
+For review: NameError, TypeError, SyntaxError, AttributeError
+
+'''
+# functions that trigger each error
+def fn_nameerror():
```

Comment:
Typically you'll want your function names to be in snake_case.  def name_error(): for example.